core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.mainpage.html

```diff
@@ -1,31 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
-  <head>
-    @load('base-components/header.template.html', {"title": "Contributor Dashboard - Oppia"})
-    <style>
-      .headroom--unpinned {
-        transform: translateY(0);
-      }
-    </style>
-  </head>
-  <body>
-    <oppia-root>
-      <div ng-controller="Base">
-        <oppia-base-content>
-          <navbar-breadcrumb>
-          </navbar-breadcrumb>
-
-          <nav-options>
-          </nav-options>
-
-          <content>
-            <contributor-dashboard-page></contributor-dashboard-page>
-          </content>
-          <page-footer></page-footer>
-        </oppia-base-content>
-      </div>
-    </oppia-root>
-    @load('pages/footer_js_libs.html')
```

Comment:
Do we need to include `footer_js_libs.html` somehow? (Ditto for header.template.html)

OR is it already added via `<page-footer></page-footer>`?

Reply:
Just as an FYI,  I found this has already been imported inside oppia-root.mainpage.html  which is rendered by the OppiaRootPage handler. So I think we can consider this resolved!